components/coding/studentPortal/ProfileDetailCard.tsx

```diff
@@ -10,57 +13,61 @@ type ProfileDetailCard = {
 function ProfileDetailCard({
   avatar,
   name,
+  link,
   joinDate,
   badges,
   currentFocus,
   nextGoal,
 }: ProfileDetailCard) {
+  let profile = "profile/" + link;
   return (
     <div className="bg-gray-900 text-sm w-60 p-6 h-full space-y-8 rounded-md ">
-      <a href="profile">
-        <div className="flex flex-col grow items-center gap-4  hover:border-2 hover:rounded-md">
-          <img
-            className="rounded-full h-full border-2 w-20 flex "
-            src={avatar}
-          />
-          <p className=" font-bold "> {name}</p>
-          <div className="flex flex-row">
-            <svg
-              xmlns="http://www.w3.org/2000/svg"
-              viewBox="0 0 24 24"
-              fill="currentColor"
-              className="w-4 h-4 text-charmander"
-            >
-              <path
-                fill-rule="evenodd"
-                d="M12 2.25c-5.385 0-9.75 4.365-9.75 9.75s4.365 9.75 9.75 9.75 9.75-4.365 9.75-9.75S17.385 2.25 12 2.25zM12.75 6a.75.75 0 00-1.5 0v6c0 .414.336.75.75.75h4.5a.75.75 0 000-1.5h-3.75V6z"
-                clip-rule="evenodd"
-              />
-            </svg>
+      <Link href={profile}>
```

Comment:
IMO a better pattern would be to wrap the Link component outside the ProfileDetailCard component in the coaches-dashboard parent. I feel like this is more intuitive because if I'm reading the code from that parent page, I know where each component links to without going into that component. I feel like I've seen this pattern more frequently

I'm curious whether this affects the difference in loading time that you noticed after switching from the <a> tag

Reply:
Wrapping the Link tag in the parent component does not seem to speed up the loading time of the link.